src/Tasks/MigrateContentToElement.php

```diff
@@ -2,13 +2,50 @@
 
 namespace DNADesign\Elemental\Tasks;
 
+use DNADesign\Elemental\Extensions\ElementalAreasExtension;
+use DNADesign\Elemental\Extensions\ElementalPageExtension;
+use DNADesign\Elemental\Models\BaseElement;
 use DNADesign\Elemental\Models\ElementalArea;
 use DNADesign\Elemental\Models\ElementContent;
-
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Dev\BuildTask;
+use SilverStripe\Versioned\Versioned;
 
 class MigrateContentToElement extends BuildTask
 {
+    /**
+     * Configures if the existing content should be left blank.
+     *
+     * @config
+     * @var bool
+     */
+    private static $clear_content = true;
+
+    /**
+     * The FQN of an element that will be the target of the content
```

Comment:
Do you want to add something like "must extend BaseElement" ?

Reply:
That's already a requirement of elements as `ElementTypeRegistry` looks for subclasses of `BaseElement`. I can add something if you feel strongly though